docs/deployments/inferentia.md

```diff
@@ -0,0 +1,84 @@
+# Using Inferentia
+
+_WARNING: you are on the master branch, please refer to the docs on the branch that matches your `cortex version`_
+
+To use [Inferentia ASICs](https://aws.amazon.com/machine-learning/inferentia/):
+
+1. You may need to [file an AWS support ticket](https://console.aws.amazon.com/support/cases#/create?issueType=service-limit-increase&limitType=ec2-instances) to increase the limit for your desired instance type.
+1. Set the instance type to an AWS Inferentia instance (e.g. `inf1.xlarge`) when creating your Cortex cluster.
+1. Set the `inf` field in the `compute` configuration for your API. One unit of `inf` corresponds to one Inferentia ASIC with 4 NeuronCores and 8GB of cache memory. Fractional requests are not allowed.
+
+## Neuron
+
+Inferentia ASICs come in different sizes depending on the instance type:
+
+* `inf1.xlarge`/`inf1.2xlarge` - each has 1 Inferentia ASIC
```

Comment:
I think it would be better to keep it as `Inferentia ASIC`. I saw that the AWS documentation refers to them as both Inferentia chips and ASICs. Calling them chips is not specific enough, it's too generic. And other providers such as Google already call them [ASICs (TPU)](https://en.wikipedia.org/wiki/Tensor_processing_unit). So, calling them by what they are (ASICs and not chips) better suites the scheme as we keep adding support for other ASICs.

Chips are CPUs too, RAM modules too, GPUs as well, SSDs as well - basically, any embedded electronic device is a chip. It's just not specific enough. ASICs on the other hand, refer to something *S* (specifically) built for some kind of application, which in this case, it's a chip specialized for ML inference.